documentation/meta/ci_cd/proof_of_functionality.md

```diff
@@ -23,7 +23,7 @@ proof-of-functionality for different use-cases.
 
 ### Catalog
 
-The tests for the catalog in `test-cat` are very comprehensive and rigourous,
+The tests for the catalog in `test-cat` are very comprehensive and rigorous,
```

Comment:
I'm a bit undecided on whether we should limit the dictionary to US English. On one hand, consistency 👍. On the other, these spellings look weird to me 😄.

Reply:
The default dictionary, IIRC, doesn't enforce US spellings in most cases. Rigourous isn't an accepted spelling anywhere, as far as I can see. Cambridge Dictionary does not mention alternative spellings for UK, at least: https://dictionary.cambridge.org/dictionary/english/rigour. Wikitionary says it's a misspelling as well, or at best an archaic version: https://en.wiktionary.org/wiki/rigourous#English

Anyway, I also agree we don't need to worry about enforcing US spellings. I would personally find it annoying to need to switch between it depending on the contexts (work vs everywhere else in my life including school), particularly when for _this_ context it matters very little if at all and in other contexts is more important.

All of this is TIL, FWIW. The French made English so confusing :rofl:

Reply:
That's my bad, I didn't look it up, just added "ous" to "rigour".

Reply:
I had to look it up too :) It's so often English defies all my expectations lol!